setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='jhhalchemy',
-    version='0.4',
+    version='0.6',
```

Comment:
Version should be 0.5 in case this gets merged first or if we decide to combine the two PRs.